### PR TITLE
Implement margins

### PR DESCRIPTION
**Pull request by Admicos**
Depends on lxqt/QTermWidget#205

### CODE REVIEW

File: src/properties.h [hunk 1]
```diff
@@ -59,6 +59,7 @@ class Properties
 
         Sessions sessions;
 
+        int margin;
```
Comment (yan12125) [inline src/properties.h]: Better to name this `terminalMargin`.

File: src/forms/propertiesdialog.ui [hunk 2]
```diff
@@ -409,6 +409,23 @@
          </property>
         </widget>
        </item>
+       <item row="17" column="0">
+        <widget class="QLabel" name="label_15">
+         <property name="text">
+          <string>Terminal margin</string>
+         </property>
+         <property name="buddy">
+          <cstring>marginSpinBox</cstring>
```
Comment (yan12125) [inline src/forms/propertiesdialog.ui]: Ditto. `terminalMarginSpinBox` is better.

File: src/forms/propertiesdialog.ui [hunk 3]
```diff
@@ -84,104 +84,7 @@
      </property>
      <widget class="QWidget" name="appearancePage">
       <layout class="QGridLayout" name="gridLayout_3">
-       <item row="8" column="0">
```
Comment (yan12125) [inline src/forms/propertiesdialog.ui]: Could you keep unrelated controls untouched? I know Qt Designer often messed up UI files, so you may need to do that manually.

**Comment by Admicos** [comment]
All of those should be fixed now. Moved the terminal margin setting to the bottom to keep my edits minimal (not that familiar with Qt stuff, that might be unneccesary)

**Comment by yan12125** [comment]
Thanks for efforts and the quick response! Merged.

**Pull request merged**